packages/mask/src/plugins/FindTruman/SNSAdaptor/hooks/useConst.ts

```diff
@@ -0,0 +1,34 @@
+import { useI18N } from '../../../../utils'
+import { fetchConst } from '../../Worker/apis'
+import { useCallback, useEffect, useState } from 'react'
+import type { FindTrumanConst } from '../../types'
+import { FindTruman_Const } from '../../constants'
+
+function replaceAll(input: string, values: Record<string, string | number>) {
+    return input.replace(/{{([^}]+)}}/g, (match, p1) => values[p1]?.toString() ?? match)
+}
```

Comment:
```suggestion
function renderString(template: string, data: Record<string, string | number>) {
    return template.replace(/{{([^}]+)}}/g, (match, p1) => data[p1]?.toString() ?? match)
}
```

Do we need to take care of multiple line string? If os, don't forget to add `m` flag, `/{{([^}]+)}}/gm`

Reply:
There's no multiple-line string at present.

Reply:
OK.